examples/pump-app/cc13x2x7_26x2x7/main/AppTask.cpp

```diff
@@ -357,6 +363,27 @@ void AppTask::DispatchEvent(AppEvent * aEvent)
     }
 }
 
+void AppTask::InitOnOffClusterState()
+{
+
+    EmberStatus status;
+
+    ChipLogProgress(NotSpecified, "Init On/Off clusterstate");
+
+    // Write false as pump always boots in stopped mode
+
+    status = OnOff::Attributes::OnOff::Set(ONOFF_CLUSTER_ENDPOINT, false);
+
+    if (status != EMBER_ZCL_STATUS_SUCCESS)
+
```

Comment:
spacing here is too large: please remove the extra newlines between every line. At least if braces should be more compact

Reply:
Corrected

Reply:
Doesn't look corrected in the code in the PR to me... Please don't resolve until the code is updated, because once the comment is resolved this will get merged as it is at that point.

Reply:
I got tricked by a our VPN to the push did not go through - It's there now.